### PR TITLE
Instructions for updating the reviewer tracking spreadsheet, per #575…

### PR DESCRIPTION
**Pull request by jerielizabeth**
… #560

Adding language on updating the google spreadsheet with reviewer requests.

Question for the group on whether we want to include the reviewer's email address for "identity" purposes (to disambiguate).

### CODE REVIEW

**Comment by jerielizabeth** [comment]
Will also need translation once we are all happy with the language. @mariajoafana @arojascastro @vgayolrs

File: editor-guidelines.md [hunk 1]
```diff
@@ -56,7 +56,19 @@ The *Programming Historian* uses a model of open peer review, while we believe t
...
+
+For each potential reviewer you do contact, regardless of response, please enter: 
+
++ the date contacted, 
++ the reviewer's name, 
++ your name as the editor, 
++ the lesson to be reviewed,
++ and the response. 
+
+You can enter the date manually (please use mm/dd/yyyy) or using the `=today()` function.
```
Comment (mdlincoln) [inline editor-guidelines.md]: Will `=today()` just keep updating the day every time the sheet is opened? (I've never actually tried this with Google Sheets before!)

**Comment by arojascastro** [comment]
Good @jerielizabeth ! How can we get access to the spreadsheet?

**Comment by jerielizabeth** [comment]
@arojascastro I just added you (sorry! I must have missed you earlier!) I have sent around a link to the google group email that will give access and I am adding people to the doc directly as they come on. Everyone should also be able to add others to the spreadsheet.

File: editor-guidelines.md (hunk 1 above)
Comment (jerielizabeth) [reply editor-guidelines.md]: I hadn't used it before either :) It appears it would keep updating. I've removed that suggestion.

**Comment by arojascastro** [comment]
No worries! I think I am not subscribed to the Google group. Can anyone add me? @jerielizabeth @acrymble

**Review by mdlincoln — APPROVED** [review]

**Comment by jerielizabeth** [comment]
Any objections to merging this one in?

**Comment by JMParr** [comment]
Nope. I have the image, and I think I loaded them in the right places, but
I hadn't had time to check it through GitHub desktop. Hectic couple of days.

**Comment by jerielizabeth** [comment]
@JMParr I think the image is related to https://github.com/programminghistorian/jekyll/pull/580. 
This one is for instructions on editing the spreadsheet. 

Although I just remembered your excellent suggestion to add a column for whether/when a reviewer submits the review. I'll add that and update the pull request.

**Comment by JMParr** [comment]
Sorry about that @jerielizabeth!

**Review by drjwbaker — APPROVED** [review]
Looks good.

**Comment by jerielizabeth** [comment]
Thanks, all. Waiting to hear from @arojascastro and the Spanish language team before merging in to keep the versions inline.

**Comment by arojascastro** [comment]
Ops, thanks for the reminder @jerielizabeth -- i'll work on it today in the evening. I had a look at the spreadsheet and I was wondering if we could add a column for translators as well. Or create another tab for them? Or only reviewers?  What do you think @vgayolrs and @mariajoafana ?

**Comment by arojascastro** [comment]
I have translated those terms. We will think about how to track translators in a different tab or spreadsheet, right? @vgayolrs @mariajoafana - so @jerielizabeth you can merge this.

**Comment by jerielizabeth** [comment]
Sounds good @arojascastro!

**Comment by jerielizabeth** [comment]
HathiTrust is apparently down today. Despite the red X of shame, I am going to merge this in :)

**Pull request merged**